model/src/cfd.rs

```diff
@@ -1469,6 +1469,22 @@ pub fn long_and_short_leverage(
     }
 }
 
+/// Decide the closing price according to role and position
+///
+/// Depending on role and position the closing price is determined from bid and ask price of the
+/// maker.
+/// Taker is long: When closing the taker sells short to close the position, i.e. the bid price
+/// applies.
+/// Taker is short: When closing the taker buys long to close the position, i.e. the
+/// ask price applies.
+/// For the maker the counter-position's price applies to match the price of the taker.
```

Comment:
I think the last sentence needs some more words. 
We only need this for the projects right? In order to show the same closing price for maker and taker we need to take the counter-position for the maker. 

Is my understanding correct?

Reply:
That is what I wanted to express and I *think* is is correct :)